client/src/GlobalStyles.js

```diff
@@ -20,8 +20,26 @@ function GlobalStyles() {
           background: ${theme.colors.backgroundprimary};
           font-family: 'Roboto', sans-serif;
         }
+        @keyframes zoomIn {
+          from {
+            opacity: 0;
+            transform: scale3d(0.3, 0.3, 0.3);
+          }
+
+          50% {
+            opacity: 1;
+          }
+        }
       `}
```

Comment:
Would always go with a dedicated animations.js file in /utils or something like that.